test/subset.js

```diff
@@ -30,7 +30,7 @@ describe('font subsetting', function() {
     });
 
     it('should re-encode variation glyphs', function(done) {
-      if (!fs.existsSync('/Library/Fonts/Skia.ttf')) return done();
+      if (!fs.existsSync('/Library/Fonts/Skia.ttf')) return this.skip();
```

Comment:
surely this should be an error, to be thrown instead, through an assertion after `openSync`? Skipping tests because the font files don't exist effectively introduces potential bugs as a side effect of removing a font by reducing test coverage.

Reply:
No, this just addresses to solve potential bugs. These are fonts that are not available in the repository for legal reasons. They are only in filesystem on OS, especially on MacOS.
If someone does not have these fonts, tests cannot fail with error message "font not found", they would never pass the tests and this is wrong behavior. Previously without existence of a file, tests passed "silently". Now is skipped it and is reported to user. This is the expected correct behavior in mocha tests.